app/controllers/WKRemoteTracingStoreController.scala

```diff
@@ -58,7 +60,10 @@ class WKRemoteTracingStoreController @Inject()(
             annotationLayerDAO.updateStatistics(annotation._id, report.tracingId, statistics)
           }
           userBox <- bearerTokenService.userForTokenOpt(report.userToken).futureBox
-          _ <- Fox.runOptional(userBox)(user => timeSpanService.logUserInteraction(report.timestamps, user, annotation))
+          trackTime = (!wkConf.WebKnossos.User.timeTrackingOnlyWithSignificantChanges ||
+            (wkConf.WebKnossos.User.timeTrackingOnlyWithSignificantChanges && report.significantChangesCount > 0))
```

Comment:
I feel like this expression can be simplified. Didn’t verify, but shouldn’t `count > 0 || (!onlySignificant)` be the same? I think it reads more nicely.